### ISSUE TITLE
Error loading mod

### ISSUE THREAD

**Issue by Sunekaer** [McJty/McJtyLib#103]
![image](https://user-images.githubusercontent.com/18720460/62563814-13451880-b884-11e9-9a38-e11619c1433a.png)
https://paste.dimdev.org/govapulozo.md
forge 28.0.45

**Comment by Siriuo** [comment]
Same issue for me.

**Comment by McJty** [comment]
Yes, don't update to forge 28.0.45. It breaks mods (JEI is also broken on that version btw)

**Comment by Sunekaer** [comment]
JEI has been update
https://www.curseforge.com/minecraft/mc-mods/jei/files/2757010

**Comment by Sunekaer** [comment]
And its still not updated for .45 -.- :D

**Comment by ProsperCraft** [comment]
Same issue.
[debug.log](https://github.com/McJtyMods/McJtyLib/files/3484599/debug.log)
[latest.log](https://github.com/McJtyMods/McJtyLib/files/3484600/latest.log)

**Comment by ratingcrib** [comment]
i have the same problem but im on the newest version of forge

**Comment by josephcsible** [comment]
@ratingcrib That's the problem. Downgrade to something below 28.0.45 for now.

**Comment by ratingcrib** [comment]
> @ratingcrib That's the problem. Downgrade to something below 28.0.45 for now.

ok thank you
